### PR TITLE
Demo event attendance [Event Attendance Part +2]

### PR DESCRIPTION
**Pull request by sleidig**
generate demo events including individual attendance for every RecurringActivity

### CODE REVIEW

File: src/app/core/config/config-fix.ts [hunk 1]
```diff
@@ -1,4 +1,8 @@
-{
+import { defaultAttendanceStatusTypes } from "./default-config/default-attendance-status-types";
+import { defaultInteractionTypes } from "./default-config/default-interaction-types";
+
+// prettier-ignore
+export const defaultConfig = {
```
Comment (TheSlimvReal) [inline src/app/core/config/config-fix.ts:5]: we should be careful to never put any logic in this file

**Review by TheSlimvReal — APPROVED** [review]
The demo data looks good, but when I am in Child Details, I can not select a different month in the calendar for the attendance. Is this intended?

**Comment by sleidig** [comment]
> when I am in Child Details, I can not select a different month in the calendar for the attendance. Is this intended?

This is not supported yet because the calendar is only showing events of the first aggregation period of the table (i.e. the most recent month). We might be able to extend this feature though. Will add this to the list for the epic

File: src/app/core/config/config-fix.ts (hunk 1 above)
Comment (sleidig) [reply src/app/core/config/config-fix.ts:5]: that's an important remark. After you raised this I looked at typescript typings for it, but it seems to be far from trivial to define a JSON type ...

**Pull request merged**